public_html/script.js

```diff
@@ -1308,6 +1473,14 @@ function onDisplayUnitsChanged(e) {
     });
 }
 
+function setAltitudeLegend(units) {
```

Comment:
Is there a way to hide the legend entirely? It takes up a lot of screen real estate